portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,39 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 90%;
 }
 
-#greeting-container {
+#quote-container {
   margin-top: 20px;
 }
+a:link {
+  color: black;
+}
+.image1 {
+  position: absolute;
+  top: 10px;
+  right: 50%;
+  width:15%; }
+
+h1 {
+  margin-left: auto;
+  margin-right: auto;
+  font-size: 4.5em;
```

Comment:
Consider combining font-size and font-family into the single font property, https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties.